bin/log-perormance-results.js

```diff
@@ -6,12 +6,27 @@
 const fs = require( 'fs' );
 const path = require( 'path' );
 const https = require( 'https' );
+const { mapKeys } = require( 'lodash' );
 const [ token, branch, hash, baseHash, timestamp ] = process.argv.slice( 2 );
 
-const performanceResults = JSON.parse(
-	fs.readFileSync(
-		path.join( __dirname, '../post-editor-performance-results.json' ),
-		'utf8'
+const resultsFiles = [
+	{
+		file: 'post-editor-performance-results.json',
```

Comment:
I was trying to run this locally. What I did was running the performance test and then move the files to where this file expects them to be. For example, for post editor:

- `npm run test:performance -- --puppeteer-interactive packages/e2e-tests/specs/performance/post-editor.test.js`
- `cp packages/e2e-tests/specs/performance/post-editor.test.results.json post-editor-performance-results.json`

And similarly for the other two. Note that I had to rename the file, so I wonder if I missed another command? I guess my question is: are these paths expected?

Other than that, the command reported numbers, so it seems fine.

Reply:
yeah, you need to run the comparison command `./bin/plugin/cli.js perf branch1 branch2` to get the right files.